config/reports.json

```diff
@@ -1038,7 +1038,7 @@
     }
   },
   "chrome-ux-report": {
-    "name": "Chrome User Experience Report",
+    "name": "Chrome User Experience",
```

Comment:
"Report" is part of the product name

Reply:
The reason I removed it is cause some of the a11y tools were (not unreasonably) complaining about the confusion of two "Chrome User Experience Report" links, linking to two difference places:

![image](https://user-images.githubusercontent.com/10931297/116013378-d0af0b00-a627-11eb-80a2-e0b39c859485.png)

The heading links to the HTTP Archive CrUX Report (The Chrome User Experience Report Report if you will!), while the second link goes to the CrUX home page.

It's a minor thing so can just revert and ignore it, unless you've any better suggestions on how to handle this?

Reply:
There's also the fact I renamed the buttons so we wouldn't have all the buttons called "View the Report", but a more descriptive button:

![image](https://user-images.githubusercontent.com/10931297/116013756-a3635c80-a629-11eb-8b04-dfe27e3ab92e.png)

If we revert "Chrome User Experience" back to "Chrome User Experience Report", then it's button will become "View the Chrome User Experience Report Report" which, while technically accurate is a bit of a mouthful!

Plus I didn't like the fact the original title wrapped when none of the others did 😁

Reply:
How about renaming the report to `CrUX`? Shorter and allows "Report" after.

Reply:
Done!